lib/reality/geo.rb

```diff
@@ -45,6 +48,28 @@ def initialize(lat, lng)
         @lat, @lng = Rational(lat), Rational(lng)
       end
 
+      def distance_to(point)
```

Comment:
For me, here is not obvious, but important use case: `Kyiv.coord.distance_to(Kharkiv)`.
I feel it _should_ work (in other words, something which `is_a?(Reality::Entity)` and `respond_to?(:coord)`). 
On other hand, `distance_to([lat, lng])` or `distance_to(latlng_str)` is not too interesting cases, maybe.